src/components/Nav.js

```diff
@@ -1,13 +1,48 @@
-import { Link } from 'react-router-dom';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaUserAlt } from 'react-icons/fa';
 
 const Nav = () => {
-  const nav = (
-    <div className="navBar">
-      <Link className="home" to="/">Books</Link>
-      <Link className="categories" to="/categories">Categories</Link>
-    </div>
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Books',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      text: 'Categories',
+    },
+  ];
+
+  return (
+    <nav className="flex justify-between py-2 bg-gray-00 items-center border-b border-b-gray-200">
+      <div className="flex space-x-14 items-center mx-5">
+
+        <h1 className="text-3xl font-bold text-blue-700">Bookstore CMS</h1>
+        <ul className="flex space-x-14 text-2l font-thin">
+          {links.map(({ id, path, text }) => (
+            <li key={id}>
+              <NavLink
+                end
+                to={path}
+                className="hover:text-zinc-50 active:text-gray-400"
+              >
+                {text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div>
+
+        <div className="bg-white p-4 rounded-full mx-5">
+          <FaUserAlt className="text-blue-600" />
```

Comment:
- [ ] Your trying to use icon from the react-icons kindly consider adding react-icons to your package.json by running `npm i react-icons` so the error is fixed

![Screenshot_20221112_222642](https://user-images.githubusercontent.com/25227022/201495274-28556ea3-4a73-4e10-ad6f-acb8645ab4a1.png)